source/_components/person.markdown

```diff
@@ -86,3 +86,15 @@ person:
       - device_tracker.stacey
       - device_tracker.beacon
 ```
+
+### Customizing the picture for a person
+
+You can customize the picture used for a person entity in the `customize:` section of your configuration. For example:
+
+```yaml
+customize:
+  person.ada6789
+    entity_picture: "/local/ada.JPG"
```

Comment:
`.JPG` is a weird file extension, please use lowercase.

Reply:
Fixed